.github/workflows/ci.yml

```diff
@@ -34,5 +34,7 @@ jobs:
         pylint mindyolo --rcfile=.github/pylint.conf
     - name: Test with unit test (UT) pytest
       run: |
+        wget https://github.com/ultralytics/yolov5/releases/download/v1.0/coco128.zip
+        unzip coco128.zip
```

Comment:
mv to test_loader.py, follow mindcv
https://github.com/mindspore-lab/mindcv/blob/main/tests/modules/test_loader.py#L60